src/project/FileTreeView.js

```diff
@@ -478,7 +478,42 @@ define(function (require, exports, module) {
         render: function () {
             var fullname = this.props.name,
                 extension = LanguageManager.getCompoundFileExtension(fullname),
-                name = _getName(fullname, extension);
+                name = _getName(fullname, extension),
+                fileType = "default";
+
+            switch (extension.toLowerCase()) {
+              case 'htmls':
```

Comment:
I'm surprised you don't have to prefix all of these with a `.` since `LanguageManager.getCompoundFileExtension(fullname)` is supposed to add the `.` to the front.